examples/all-clusters-app/esp32/main/Kconfig.projbuild

```diff
@@ -103,10 +103,11 @@ menu "Demo"
 endmenu
 
 menu "PW RPC Debug channel"
+depends on ENABLE_PW_RPC
     config EXAMPLE_UART_PORT_NUM
         int "UART port number"
         range 0 2 if IDF_TARGET_ESP32
```

Comment:
I think there are 2 UART ports in ESP32-C3 and hence the range should be  0 1 (at all places)

Reply:
Done